addon/components/column-chart/component.js

```diff
@@ -64,7 +64,8 @@ export default BaseChartComponent.extend({
         }
 
         if (this.get('yAxis') && this.get('yAxis').domain) {
```

Comment:
```suggestion
        if (this.get('yAxis.domain')) {
```

Can shorten this by using ember get's safe path lookup